plugins/GSdx/GSDevice11.cpp

```diff
@@ -307,7 +307,7 @@ bool GSDevice11::Create(GSWnd* wnd)
 
 	memset(&sd, 0, sizeof(sd));
 
-	sd.Filter = sd.Filter = !!theApp.GetConfig("AnisotropicFiltering", 0) && !theApp.GetConfig("paltex", 0) ? D3D11_FILTER_ANISOTROPIC : D3D11_FILTER_MIN_MAG_MIP_LINEAR;
+	sd.Filter = sd.Filter = theApp.GetConfig("MaxAnisotropy", 1)!=1 && !theApp.GetConfig("paltex", 0) ? D3D11_FILTER_ANISOTROPIC : D3D11_FILTER_MIN_MAG_MIP_LINEAR;
```

Comment:
this would be better than changing the value of m_gs_max_anisotropy.push_back(GSSetting(1, "1", "off"));. to  m_gs_max_anisotropy.push_back(GSSetting(0, "Off", "")); since 0 is not a valid range at this condition and won't be accepted even if users customize through the ini. any opinions ? @gregory38